scripts/proof_bot.py

```diff
@@ -118,6 +118,7 @@ def get_and_push_proof(
         )
 
         if response.status_code != 200:
+            logger.debug(response)
             logger.warning("Skipping transaction " + txhash)
```

Comment:
I think response by itself just looks something like "<Response [504]>" for whatever the status code is.

Perhaps the following, wdyt?
```suggestion
        if response.status_code != 200:
            logger.warning(f"[Error {response.status_code} - {response.text}]: Skipping transaction {txhash}")
```

Reply:
✔️